src/maintainer/conda_forge_yml.rst

```diff
@@ -79,6 +79,9 @@ mapping for Azure-specific configuration options. For example:
       # toggle for freeing up some extra space on the default Azure Pipelines
       # linux image before running the Docker container for building
       free_disk_space: False
+      # toggle for creating a 10G swap file on the default Azure Pipelines
+      # linux image before running the Docker container for building
+      make_10G_swap: False
```

Comment:
Wonder if we should allow the size to be customizable. Something like this perhaps?

```suggestion
      make_swap_file: 10Gb
```

Reply:
Yes this is a great idea, the default can be zero and if it is zero it just skips that whole block, but if greater than zero we make a swap file of that size

Reply:
Yeah actually default of `0` makes more sense. Let's go with that